geopandas/tests/test_plotting.py

```diff
@@ -725,6 +782,12 @@ def test_polygons_values(self):
         _check_colors(self.N, coll.get_edgecolor(), ["g"] * self.N)
         ax.cla()
 
+        # handle None
+        coll = plot_polygon_collection(ax, self.polygons, [0, None, 3])
```

Comment:
I personally don't think we need to support this. This was working only accidentally before, and was ignoring all valid values as well. 
I would consider it a bug fix that this now starts to raise an error instead of silently being ignored?

We need better support for missing values in the `values`, for sure, see eg https://github.com/geopandas/geopandas/issues/695 and https://github.com/geopandas/geopandas/issues/402

Reply:
Okay, I don't mind. I'll remove this support from this PR and let Matplotlib raise the error. I'll try to have a look at the support of missing values soon, so this should not be an issue for long.